sentry.gradle

```diff
@@ -92,7 +92,10 @@ gradle.projectsEvaluated {
               } catch (FileNotFoundException e) {
                   project.logger.info("file not found '$propertiesFile' for '$variant'")
               }
-              def cliExecutable = sentryProps.get("cli.executable", "$reactRoot/node_modules/@sentry/cli/bin/sentry-cli")
+
+              def resolvedCliPackage = new File(["node", "--print", "require.resolve('@sentry/cli/package.json')"].execute(null, rootDir).text.trim()).getParentFile();
```

Comment:
I believe this could throw in a few different ways, ideally, we'd wrap in a try-catch, so the plugin keeps running in case it goes wrong.